mantidproject/mantid
# Fix performance tests

**Description of work.**

Fixes [master_performancetests](https://builds.mantidproject.org/job/master_performancetests2/lastCompletedBuild/testReport/). One algorithm, LoadVulcanCalFile, is obsolete as confirmed by SNS and usage stats and has been deleted.

**To test:**

Run the `FindPeaksTestPerformance` test and see that it passes.

Fixes #27059

*This does not require release notes* because **it relates to internal testing.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
